### PR TITLE
Add some random findings

### PR DESCRIPTION
**Pull request by LeonMrBonnie**
Just some random param types I found while researching.
All found in 2372 code

### CODE REVIEW

File: natives.json [hunk 1]
```diff
@@ -56877,11 +56877,11 @@
 			"comment": "",
 			"params": [
 				{
-					"type": "Any",
+					"type": "int",
 					"name": "p0"
 				},
 				{
-					"type": "Any*",
+					"type": "int*",
```
Comment (alloc8or) [inline natives.json:56884]: This is a gamer/network handle (a struct) so it should stay ``Any*``. I've seen too much bad code where people incorrectly use gamer handle stuff due to the params often being ``int*``.

File: natives.json [hunk 2]
```diff
@@ -57678,31 +57678,31 @@
...
+					"type": "const char*",
 					"name": "p4"
 				},
 				{
 					"type": "BOOL",
 					"name": "p5"
 				}
 			],
-			"return_type": "Any",
+			"return_type": "Any*",
```
Comment (alloc8or) [inline natives.json:57705]: This is actually an integer (a hash I think).

**Review by alloc8or — APPROVED** [review]

**Comment by alloc8or** [comment]
Oops, looks like I clicked the wrong thing. Requesting changes for the two things above obviously.

File: natives.json (hunk 1 above)
Comment (LeonMrBonnie) [reply natives.json:56884]: My bad. I took a look again and this was an error from my side.

File: natives.json (hunk 2 above)
Comment (LeonMrBonnie) [reply natives.json:57705]: Can you maybe give me some insight on why this is actually an integer?
The decompiled source always uses the native like this:
```c
´*uParam0 = NETWORK::UGC_TEXTURE_DOWNLOAD_REQUEST(sParam1, iParam4, iParam2, 0, sParam1, false);
uParam0->f_1 = 2;
```
So from my understanding it has to be a struct.

Comment (alloc8or) [reply natives.json:57705]: The decompiler doesn't have the nicest output for struct fields at offset `0`.
That `*uParam0 = ...` here is actually `*uParam0.f_0 = ...`, which is an integer (and seems to be a hash from what I can see in game code).

Comment (LeonMrBonnie) [reply natives.json:57705]: Ah okay, makes sense then.

**Pull request merged**